### ISSUE TITLE
How do you built your sensor?

### ISSUE THREAD

**Issue by RemoPro** [sighmon/ios-ble-co2-sensor#1]
You list the components but not how to connect these to build a sensor myself

**Comment by sighmon** [comment]
@RemoPro I've added instructions here: https://github.com/sighmon/ios-ble-co2-sensor#assembly

Let me know if you'd like extra information.

**Comment by RemoPro** [comment]
Does the sensor function with Bluetooth everywhere after the initial setup? So I can use it if I have no wifi

**Comment by sighmon** [comment]
@RemoPro Yes it works without an internet connection. It only needs WiFi if you'd like to upload your readings to an influxdb cloud account.

**Comment by RemoPro** [comment]
Great, how much is the total hardware cost?

**Comment by sighmon** [comment]
@RemoPro Depends where you live - for me in Australia ~AUD$150 without the battery and charger.

**Comment by RemoPro** [comment]
I only need the battery charger when I want to use an battery right?

**Comment by sighmon** [comment]
@RemoPro Correct. Else connecting it to a USB-C power source will work.
